cpp/src/search/contains_nested.cu

```diff
@@ -63,4 +65,24 @@ bool contains_nested_element(column_view const& haystack,
            }) > 0;
 }
 
+std::unique_ptr<column> multi_contains_nested_elements(column_view const& haystack,
+                                                       column_view const& needles,
+                                                       rmm::cuda_stream_view stream,
+                                                       rmm::mr::device_memory_resource* mr)
+{
+  auto result_v = contains(table_view{{haystack}},
+                           table_view{{needles}},
+                           null_equality::EQUAL,
+                           nan_equality::ALL_EQUAL,
+                           stream,
+                           mr);
+
+  // todo: https://github.com/rapidsai/cudf/pull/11356
+  auto result =
+    std::make_unique<column>(data_type{type_to_id<bool>()}, needles.size(), result_v.release());
+  result->set_null_mask(copy_bitmask(needles, stream, mr), needles.null_count());
```

Comment:
Can do these two lines in a single call to the ctor?
```suggestion
  auto result =
    std::make_unique<column>(data_type{type_to_id<bool>()}, needles.size(), result_v.release(),
                             copy_bitmask(needles, stream, mr), needles.null_count());
```

Reply:
Sure. However, I want to wait for this: https://github.com/rapidsai/cudf/pull/11356#issuecomment-1199944110